client/packages/system/src/Vaccination/utils.ts

```diff
@@ -5,18 +5,35 @@
 import { VaccinationDraft } from './api';
 import { VaccinationCardItemFragment } from './api/operations.generated';
 
-export const isPreviousDoseGiven = (
+export const getPreviousDoseStatus = (
```

Comment:
Needed to check what the previous status actually is to determine how the modal should display. Can use this value to also do the same thing `isPreviousGiven` does, so have removed that function.